data/data2b.s

```diff
@@ -4509,7 +4510,7 @@ gUnknown_08305D0C:: @ 8305D0C
 @ 8305DCC
 	.include "data/enemy_mon_elevation.inc"
 
-@ 8305F68
+@ 8309AAC
```

Comment:
Why did you change this? It's definitely not located there.

Reply:
fixed